tests/components/shelly/test_config_flow.py

```diff
@@ -706,6 +706,30 @@ async def test_zeroconf_already_configured(hass):
     assert entry.data["host"] == "1.1.1.1"
 
 
+async def test_zeroconf_ignored(hass):
+    """Test zeroconf when the device was previously ignored."""
+
+    entry = MockConfigEntry(
+        domain="shelly",
+        unique_id="test-mac",
+        data={},
+        source=config_entries.SOURCE_IGNORE,
+    )
+    entry.add_to_hass(hass)
+
+    with patch(
+        "aioshelly.common.get_info",
```

Comment:
This patch target should probably be `"homeassistant.components.shelly.config_flow.aioshelly.common.get_info"` but all the others were this way so I didn't want to update it in this PR as it would make the PR larger than the scope of the fix

Reply:
Maybe we should make a fixture for this as it repeats in multiple tests